src/Department.java

```diff
@@ -1,7 +1,24 @@
 package application;
 
 public class Department {
-	public boolean isEquals(Department other) {
-		return false;
+	String name="Test";
+	int id=123;
+	String country="Neverland";
+	String city="Gotham";
```

Comment:
Put this in a Constructor

~~~java
public Department() {
    name = "Test";
    id = 123;
    country = "Neverland";
    city = "Gotham";
}
~~~